CMakeLists.txt

```diff
@@ -150,6 +150,6 @@ add_wfa_test()
 
 install(TARGETS wfa ARCHIVE DESTINATION lib)
 
-install(FILES ${INCLUDES} DESTINATION include)
+install(FILES ${INCLUDES} DESTINATION include/wfa2lib)
```

Comment:
Ah, I *was* worried we'd have multiple iterations :). Looks like an easy fix.

Reply:
Well, that was the official documentation for GNUInstallDirs, but I probably shouldn't have assumed that using it is obvious.

(I really wish CMake was a bit more like Meson. In Meson, if you create a library you just set the `install: true` attribute and it automatically defaults to installing into the user configured libdir if you don't manually specify a destination.)

Reply:
Yeah, thank you for helping out. I'll apply this to other CMake projects we have. CMake I have to treat like a dungeon game and it is very ugly as a language. I have written CMake modules/extensions a long time ago and that was no joy. I use meson in another project and, even though it looks simpler, I find it harder to trouble shoot on edge cases. Maybe it is just lack of experience. The world could use a great functional build system ;)

Reply:
> I use meson in another project and, even though it looks simpler, I find it harder to trouble shoot on edge cases.

Maybe it would make sense to open a Meson issue about this?

(I don't want to go *too* off-topic in this PR. :) But I am very interested in making Meson easier to use, edge cases or not.)